packages/flutter_tools/test/general.shard/plugins_test.dart

```diff
@@ -1073,6 +1130,112 @@ flutter:
         ProcessManager: () => FakeProcessManager.any(),
       });
 
+      testUsingContext('Injects user selected implementation despite inline implementation', () async {
```

Comment:
How about "user-selected implementation overrides inline implementation"

Reply:
I wanted to conform the description in the `dart_plugin_test.dart`, where the same is tested, but I can change the it for both then. Makes sense to have the keyword `override` in it.

Reply:
Done in e637632102c53518f11babf7e77dd2cb7d21c510 .